mappings/net/minecraft/server/WorldStem.mapping

```diff
@@ -2,7 +2,9 @@ CLASS net/minecraft/unmapped/C_tljyhzqp net/minecraft/server/WorldStem
 	FIELD f_hmwuaxvg saveProperties Lnet/minecraft/unmapped/C_jkevluio;
 	FIELD f_sphgqqzg resourceManager Lnet/minecraft/unmapped/C_carvbxzj;
 	FIELD f_twvealqc resources Lnet/minecraft/unmapped/C_migzkpst;
+	FIELD f_yskbtpkd registries Lnet/minecraft/unmapped/C_bcpxdrik;
 	METHOD close close ()V
-	METHOD m_pfrxpzhv resources ()Lnet/minecraft/unmapped/C_migzkpst;
-	METHOD m_vmpnykme resourceManager ()Lnet/minecraft/unmapped/C_carvbxzj;
-	METHOD m_vqetafdc saveProperties ()Lnet/minecraft/unmapped/C_jkevluio;
+	METHOD m_pfrxpzhv getResources ()Lnet/minecraft/unmapped/C_migzkpst;
```

Comment:
this is a record, so we don't use the `get` name. please revert the changes to this class!

Reply:
oh oops, my bad